src/components/navbar/Navbar.css

```diff
@@ -0,0 +1,52 @@
+.navbar-icon {
```

Comment:
Is this CSS file automatically created when you create a SCSS file?

Reply:
There is an extension called Live Sass compiler that will change scss files to css.